src/client/main.cpp

```diff
@@ -2,6 +2,87 @@
 #include <memory>
 #include <string>
 
+#include "helper/parser.h"
+
+using namespace std;
+
+string loggedInUsername = "";
+
+void handleCreateHost(vector<string> input) {
+  if (input.size() != 3) {
+    cout << "Invalid input for creating account.";
+  } else {
+    string username = input.at(1);
+    string password = input.at(2);
+    // This is where request to CROW(/create) is made
+    // if validated by server
+    loggedInUsername = username;
+    cout << "Successfully logged into " << loggedInUsername << ".\n";
+  }
+};
+
+void handleLoginHost(vector<string> input) {
+  if (input.size() != 3) {
+    cout << "Invalid input for logging into account.";
+  } else {
+    string username = input.at(1);
+    string password = input.at(2);
+    // This is where request to CROW(/login) is made
+    // if validated by server
+    loggedInUsername = username;
+    cout << "Successfully logged into " << loggedInUsername << ".\n";
+  }
+};
+
+void displayHelp() {
+  cout << "Commands available:";
```

Comment:
You''l want newline characters, either \n or std::endl, I prefer \n since std::endl generally flushes the output buffer, which is slower, although in this case it will have little impact.

Reply:
added.